app/browser/api/ledger.js

```diff
@@ -1702,6 +1706,15 @@ const onCallback = (state, result, delayTime) => {
     return state
   }
 
+  const newAddress = result.getIn(['properties', 'wallet', 'addresses', 'BAT'])
+  const oldAddress = ledgerState.getInfoProps(state).getIn(['addresses', 'BAT'])
+
+  console.log(newAddress, oldAddress)
```

Comment:
whoops- looks like you left a console log in there 😄

Reply:
console removed